Templates/Ros2FleetRobotTemplate/README.md

```diff
@@ -0,0 +1,142 @@
+
+Refer to the [O3DE System Requirements](https://www.o3de.org/docs/welcome-guide/requirements/) documentation to make sure that the system/hardware requirements are met. 
+This project has the following dependencies:
+
+- [O3DE](https://github.com/o3de/o3de)
+- [ROS2 Humble](https://www.ros.org)
+- [ROS2 Gem](https://github.com/o3de/o3de-extras/tree/development/Gems/ROS2)
+- [Proteus robot](https://github.com/o3de/o3de-extras/tree/development/Gems/ProteusRobot)
+- [WarehouseAssets](https://github.com/o3de/o3de-extras/tree/development/Gems/WarehouseAssets)
+
+Please make sure that `clang` was installed and configured. For details refer to [this section](https://www.o3de.org/docs/welcome-guide/requirements/#linux) of O3DE documentation.
+
+## Setup Instructions
+
+The following steps will assume the following:
+
+- You have ROS2 humble [installed](https://docs.ros.org/en/humble/Installation.html) and environment is [sourced](https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Configuring-ROS2-Environment.html#source-the-setup-files).
+- The O3DE gems, projects and templates will be placed in the O3DE home folder: `/home/${USER}/O3DE`.
+- You have the O3DE engine [downloaded and built](https://www.o3de.org/docs/welcome-guide/setup/setup-from-github) and it is located in `/home/${USER}/O3DE/Engines/Development`.
+
+In this tutorial, CLI tools will be used. It is also possible to use O3DE GUI to set up a project from template. See the [O3DE Project manager documentation](https://www.o3de.org/docs/user-guide/project-config/project-manager/) for more details. In such cases it is required to source your ROS2 distro before launching O3DE manager:
+
+```shell
+source /opt/ros/humble/setup.bash
+./build/linux/bin/profile/o3de
+```
+
+### 1. Install ROS2 packages
+
+```shell
+sudo apt install ros-${ROS_DISTRO}-ackermann-msgs ros-${ROS_DISTRO}-gazebo-msgs ros-${ROS_DISTRO}-control-toolbox
+```
+
+### 2. Download the template and asset gems
+
+For convenienience, we'll define a shell variables with O3DE folders:
+
+```shell
+export O3DE_HOME=${HOME}/O3DE
+```
+
+Clone the `o3de-extras` repository containing the template and asset gems
+
+```shell
+mkdir -p ${O3DE_HOME}/Projects
+cd ${O3DE_HOME}/Projects
+git clone https://github.com/o3de/o3de-extras.git 
+cd o3de-extras 
+git lfs install && git lfs pull
+```
+
+Copy gems and template to the O3DE home.
+
+```shell
+mkdir -p ${O3DE_HOME}/Gems
+mkdir -p ${O3DE_HOME}/Templates
+cp -r ${O3DE_HOME}/Projects/o3de-extras/Gems/ROS2 ${O3DE_HOME}/Gems \
+cp -r ${O3DE_HOME}/Projects/o3de-extras/Gems/WarehouseAssets ${O3DE_HOME}/Gems \
+cp -r ${O3DE_HOME}/Projects/o3de-extras/Gems/ProteusRobot ${O3DE_HOME}/Gems \
+cp -r ${O3DE_HOME}/Projects/o3de-extras/Templates/Ros2FleetRobotTemplate ${O3DE_HOME}/Templates/
```

Comment:
Why are we making copies for these folders in the instructions? We are able to register the paths from any location